repository/src/main/java/uk/gov/hmcts/ccd/definition/store/repository/model/WizardPageComplexFieldOverride.java

```diff
@@ -55,4 +56,13 @@ public String getShowCondition() {
     public void setShowCondition(String showCondition) {
         this.showCondition = showCondition;
     }
+
+    @JsonProperty("display_context_parameter")
+    public String getDisplayContextParameter() {
+        return displayContextParameter;
+    }
+
+    public void setDisplayContextParameter(String displayContextParameter) {
+        this.displayContextParameter = displayContextParameter;
+    }
```

Comment:
not 100% sure this is needed. We already use display context parameter in Production for event fields. How do we do that at the moment? Can you check with @rafalkalita please. 
Or perhaps this would be a good improvement

Reply:
sorry ignore this this is for the Override I didn't notice that